gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -572,6 +572,19 @@ case class TransformPostOverrides(session: SparkSession, isAdaptiveContext: Bool
       p.withNewChildren(children)
   }
 
+  def nativeSupportedPlan(child: SparkPlan): Boolean = {
```

Comment:
Can you move this logic to `nativeConversion.doValidate()`

Reply:
+1